gunfolds/estimation/linear_model.py

```diff
@@ -709,7 +709,7 @@ def data2AB(data, x0=None):
     B = np.ones((n, n))
     np.fill_diagonal(B, 0)
     B[np.triu_indices(n)] = 0
-    K = np.int(np.sum(abs(B)))  # abs(A)+abs(B)))
+    K = int(np.sum(abs(B)))  # abs(A)+abs(B)))
```

Comment:
please make this `np.int` because python's int is infinite precision and I am not sure we want to deal with it